docs/demos/first_steps.md

```diff
@@ -11,17 +11,17 @@ release: 0.8.0
 # Open Service Mesh Demo: First Steps
 
 
-> Note: This demo is specific to OSM v0.8.0
+> Note: This demo is specific to OSM v0.9.0
```

Comment:
Overall I think demo instructions that live in this repo should always refer to the current commit. That would involve removing instructions for downloading the latest release and only including instructions to build from source. It seems like the docs site is better suited for instructions pertaining to a particular version of OSM (which may or may not be current).

Those kinds of changes are out of scope here though and probably need more discussion. 

fyi @bridgetkromhout